game/src/sandbox/speed.rs

```diff
@@ -595,9 +600,10 @@ impl State for TimeWarpScreen {
                     now.ampm_tostring(),
                     self.target.ampm_tostring()
                 )),
+                Line(format!("Elapsed simulation time: {}", elapsed_sim_time)),
                 Line(format!(
-                    "Elapsed simulation time: {}",
-                    Duration::realtime_elapsed(self.started)
+                    "Speed: {}x speed",
```

Comment:
How about `{:.1f}` or maybe even no decimal places? And ideally even commas every 3rd place for those maaassive speedups; so maybe `abstutil::prettyprint_usize((elapsed_sim_time / elapsed_wall_time) as usize)`?